internal/manifest/version.go

```diff
@@ -289,15 +297,26 @@ func (v *Version) Overlaps(
 					end = largest
 					restart = true
 				}
-				if restart {
-					ret = ret[:0]
-					continue loop
+			}
+
+			if restart {
+				// Need to retry the files we have not selected.
+				restart = false
+			} else {
+				// TODO: is there a better way to reserve numSelected in ret?
+				ret = append(ret, make([]FileMetadata, numSelected)...)
+				j := 0
+				for i, selected := range selectedIndices {
+					if (selected) {
```

Comment:
Nit: no need for the parentheses around `selected` in Go.

Reply:
done. Is there any tool I should be running that will catch stylistic issues (long lines, inappropriate tabs, whitespace at end etc.)